raiden-dapp/src/components/dialogs/UdcDepositDialog.vue

```diff
@@ -135,7 +135,17 @@ export default class UdcDepositDialog extends Vue {
   }
 
   get valid(): boolean {
-    return /^[1-9]\d*$/.test(this.defaultUtilityTokenAmount);
+    const utilityTokenAmount = Number(this.defaultUtilityTokenAmount);
```

Comment:
It's not safe to convert these BigNumbers to Numbers directly. They can go up to 256bits in length, and JS vms only support up to 53 bits. Please, use `parseUnits` from `ethers/utils`, with the respective decimals, and compare it using BigNumber functions.